src/constants.ts

```diff
@@ -1,6 +1,14 @@
+import { Hex } from './types';
+
 export const API_BASE_URL = 'https://transaction.metaswap.codefi.network';
-export const CHAIN_IDS = {
+export const CHAIN_IDS: {
```

Comment:
Does it work to let TypeScript infer the type via `as const` instead of manually providing the type?

Reply:
yes it does 👍

Reply:
fixed here: https://github.com/MetaMask/smart-transactions-controller/pull/210/commits/d644c5ca75e23be8e43c3a59869f7ad381db4a29